docs/resources/cbr_policy.md

```diff
@@ -0,0 +1,98 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+---
+
+## Example Usage
+
+### backup policy
+
+```hcl
+variable "policy_name" {}
+
+resource "huaweicloud_cbr_policy" "test" {
+  name                    = var.policy_name
+  operation_type          = "backup"
+  retention_duration_days = 20
+
+  trigger_pattern {
+    frequency    = "WEEKLY"
+    days         = "MO,TH"
+    backup_times = ["06:00"]
+  }
+}
+```
+
+### replication policy
+
+```hcl
+variable "policy_name" {}
+variable "project_name" {}
+variable "project_id" {}
+
+resource "huaweicloud_cbr_policy" "test" {
+  name                     = var.policy_name
+  operation_type           = "replication"
+  destination_project_name = var.project_name
+  destination_project_id   = var.project_id
+  max_backups              = 20
+
+  trigger_pattern {
+    frequency    = "DAILY"
+    interval     = 5
+    backup_times = ["21:00"]
+  }
+}
+```
+
+## Argument Reference
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CBR policy.
+  If omitted, the provider-level region will be used.
+  Changing this will create a new policy.
+
+* `name` - (Required, String) Specifies a unique name of the CBR policy.
+  This parameter can contain a maximum of 64 characters, which may consist of chinese charactors, letters, digits,
+  underscores(_) and hyphens (-).
+
+* `enabled` - (Required, Bool) Specifies whether to enable the CBR policy. Default is `true`.
+
+* `operation_type` - (Required, String, ForceNew) Specifies the CBR policy type.
+  Valid values are `backup` and `replication`.
+  Changing this will create a new policy.
+
+* `trigger_pattern` - (Required, List) Specifies the time rule for the CBR policy execution.
+  The trigger_pattern structure is documented below.
+
+* `destination_project_name` - (Optional, String) Specifies the name of the replication destination project, which is
+  mandatory for cross-region replication.
+  Required if `operation_type` is `replication`.
+
+* `destination_project_id` - (Optional, String) Specifies the ID of the replication destination project, which is
+  mandatory for cross-region replication.
+  Required if `operation_type` is `replication`.
+
+* `max_backups` - (Optional, Int) Specifies the maximum number of retained backups.
+  The value can be -1 or ranges from 2 to 99999.
+  This parameter and `retention_duration_days` are alternative.
+
+* `retention_duration_days` - (Optional, Int) Specifies the duration (in days) for retaining backups.
+  The value ranges from 0 to 99999.
```

Comment:
the value range is different with the schema definition, please check this.

Reply:
the wrong document description has been modified.